rust/connlib/tunnel/src/device_channel.rs

```diff
@@ -85,10 +85,25 @@ impl Device {
     #[cfg(target_family = "windows")]
     pub(crate) fn poll_read<'b>(
         &self,
-        _: &'b mut [u8],
-        _: &mut Context<'_>,
+        buf: &'b mut [u8],
+        cx: &mut Context<'_>,
     ) -> Poll<io::Result<Option<MutableIpPacket<'b>>>> {
-        Poll::Pending
+        // TODO: I don't understand the MTU slice here
+        // Are we doing partial reads so that the IP packets can fragment if needed instead of dropping?
```

Comment:
it's just that we should never read more than `mtu`

Reply:
Okay. I figured since wintun and connlib are both dealing in whole packets, if I slice a packet at MTU length, that means some data goes missing.

I'll remove the comment and assume someone else understands it - I'm just following the unix impl